### PR TITLE
samples: Bluetooth: Mesh: increase HCI cmd buffers

### PR DESCRIPTION
**Pull request by akredalen**
Increased the number of HCI command buffers for samples with additional SMP services.
One additional buffer per service added.

### CODE REVIEW

**Review by PavelVPV — APPROVED** [review]

File: samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf [hunk 1]
```diff
@@ -61,3 +61,6 @@ CONFIG_BT_MESH_ONOFF_SRV=y
 # Enable the LBS service
 CONFIG_BT_LBS=y
 CONFIG_BT_LBS_POLL_BUTTON=y
+
+# Extra HCI command buffers for SMP service
+CONFIG_BT_BUF_CMD_TX_COUNT=4
```
Comment (omkar3141) [inline samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf:66]: Does each service requires additional command buffer?

File: samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf [hunk 2]
```diff
@@ -61,3 +61,6 @@ CONFIG_BT_MESH_ONOFF_SRV=y
 # Enable the LBS service
 CONFIG_BT_LBS=y
 CONFIG_BT_LBS_POLL_BUTTON=y
+
+# Extra HCI command buffers for SMP service
```
Comment (m-alperen-sener) [inline samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf:65]:
```suggestion
# Increasing the HCI command buffers to 4 for additional Peripheral LE Service
```

File: samples/bluetooth/mesh/light/overlay-dfu.conf [hunk 3]
```diff
@@ -18,3 +18,6 @@ CONFIG_BT_MAX_CONN=3
 
 # One extra identity for SMP service
 CONFIG_BT_ID_MAX=2
+
+# Extra HCI command buffers for SMP service
```
Comment (m-alperen-sener) [inline samples/bluetooth/mesh/light/overlay-dfu.conf:22]:
```suggestion
# Increasing the HCI command buffers to 4 for additional SMP service
```

File: samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf (hunk 1 above)
Comment (akredalen) [reply samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf:66]: The number 4 was based of an earlier [PR](https://github.com/nrfconnect/sdk-nrf/pull/19090/files). One per service should be right. Let me check if either of these samples need less.
Also, this is documented [here](https://github.com/zephyrproject-rtos/zephyr/blob/5dfd41e51a86c75d156241b7566ad4c1c1dce2d5/subsys/bluetooth/mesh/Kconfig#L128) but only for `BT_MESH_WORKQ_SYS` and not for `BT_MESH_WORKQ_MESH` which is default (and chosen for these samples)

Comment (akredalen) [reply samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf]: Only 3 was needed for these samples. I guess both of them only adds one additional service. Updated now.

Comment (akredalen) [reply samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf]: Any last comments about the BT_MESH_WORKQ_MESH doc as mentioned above?

**Review by m-alperen-sener — APPROVED** [review]

File: samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf (hunk 1 above)
Comment (m-alperen-sener) [reply samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf]: I think it is OK as it is now.

**Pull request merged**